File: Source/JapaneseLanguageTools.Contracts/Models/Responses/GetWordExerciseBatchResponse.Model.cs
using JapaneseLanguageTools.Contracts.Models.Requests;

namespace JapaneseLanguageTools.Contracts.Models.Responses;

public class GetWordExerciseBatchResponseModel
{
    public required WordExerciseBatchModel WordExerciseBatch { get; set; }

    public required GetWordExerciseBatchRequestModel Request { get; set; }
}

File: Source/JapaneseLanguageTools.ReactSpa/source/models/responses/GetWordExerciseBatchResponse.Model.ts
import WordExerciseBatchModel from "@/models/WordExerciseBatch.Model";
import GetWordExerciseBatchRequestModel from "@/models/requests/GetWordExerciseBatchRequest.Model";

export default interface GetWordExerciseBatchResponseModel {
  wordExerciseBatch: WordExerciseBatchModel;

  request: GetWordExerciseBatchRequestModel;
}
